Sources/SmartInsights/UpdateAvailableInsight.swift

```diff
@@ -11,6 +11,7 @@ import Combine
 
 /// Uses the bundle identifier to fetch latest version information and provides insights into whether
 /// an app update is available.
+@available(iOS 13, *)
```

Comment:
Would you like to have these insights as well? I'd say we could consider not using Combine so that it can be supported on iOS 11+, too.

Reply:
This is a great suggestion. Let me change the UpdateAvailableInsight to not using Combine. As far as I can see, it is a simple network call to fetch app info from App Store

Reply:
Let us know if you want to tackle this in this pull request or after! :)

Reply:
let me do this in this MR. A little bit busy at work now but I'll update the MR soon 🙂

Reply:
All good, take your time! :)

Reply:
@BasThomas I've finally found some time to convert combine api to completion-based API to support iOS 11.

Please let me know if you still find something to improve.